dev-vcs/git-pw/git-pw-2.0.0.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{8..9} )
+DISTUTILS_USE_SETUPTOOLS=bdepend
+
+inherit distutils-r1
+if [[ "${PV}" == 99999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/getpatchwork/git-pw.git"
+else
+	SRC_URI="https://github.com/getpatchwork/git-pw/archive/${PV}.tar.gz -> $P.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+DESCRIPTION="A tool for integrating Git with Patchwork"
+HOMEPAGE="https://github.com/getpatchwork/git-pw"
+
+LICENSE="MIT"
+SLOT="0"
+IUSE=""
+
+DEPEND="
+	dev-python/arrow[${PYTHON_USEDEP}]
+	dev-python/click[${PYTHON_USEDEP}]
+	dev-python/requests[${PYTHON_USEDEP}]
+	dev-python/tabulate[${PYTHON_USEDEP}]
+"
+
+RDEPEND="${DEPEND}"
+BDEPEND=""
+
```

Comment:
Try running [tests](https://dev.gentoo.org/~mgorny/python-guide/distutils.html#enabling-tests)?

Reply:
not sure how to handle this? any example?

Reply:
This is a nice [example](https://gitweb.gentoo.org/repo/gentoo.git/tree/dev-python/python-utils/python-utils-2.5.6.ebuild#n19)